### PR TITLE
DM-29546: Add two buckets to panda-dev

### PR DESCRIPTION
**Pull request by hsinfang**

### CODE REVIEW

File: environment/deployments/panda/env/dev.tfvars [hunk 1]
```diff
@@ -133,4 +133,5 @@ image         = "centos-7-v20210316"
 tags          = ["allow-ssh"]
 type          = "pd-standard"
 # source_image_family  = "centos-7"
-# source_image_project = "centos-cloud"
\ No newline at end of file
+# source_image_project = "centos-cloud"
```
Comment (timj) [inline environment/deployments/panda/env/dev.tfvars:136]: This change should probably be on a separate commit.

**Review by timj — APPROVED** [review]
This looks like what we agreed.

File: environment/deployments/panda/env/dev.tfvars (hunk 1 above)
Comment (hsinfang) [reply environment/deployments/panda/env/dev.tfvars:136]: I did this change only to trigger the GitHub Action pipeline
